### PR TITLE
OC-13298 https://jira.openclinica.com/browse/OC-13298

### PR DESCRIPTION
**Pull request by jkeremian**
Audit log is not preserving old value and new value in some scenarios in ODM after migration

### CODE REVIEW

File: src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java [hunk 1]
```diff
@@ -222,7 +222,7 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                     }
 
                     if (BooleanUtils.isTrue(se.getLocked()))
-                        xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("locked"));
+                        xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("Locked"));
                     else if (BooleanUtils.isTrue(se.getSigned()))
                         xml.append("\" OpenClinica:Status=\"" + StringEscapeUtils.escapeXml("signed"));
```
Comment (svadla-oc) [inline src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java:227]: is Signed supposed to start with uppercase letter too?

Comment (jkeremian) [reply src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java:227]: It was not reported as an issue the signed one.

Comment (jkeremian) [reply src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java:227]: Jigyasa says only Locked ones and not the signed ones.

**Comment by svadla-oc** [comment]
👍

**Pull request merged**